test/pkg/require/requires.go

```diff
@@ -11,7 +11,7 @@ import (
 // Features checks the wanted features are present in given,
 // and skips the test if any are missing. If the given set is empty,
 // the function returns (by default all features are enabled).
-func Features(tb testing.TB, given *flag.StringSet, want ...string) {
+func Features(tb testing.TB, given *flag.ExcludeStringSet, want ...string) {
```

Comment:
Update comments?

Reply:
What are you looking for in the comment? I dont think it needs to be updated.

Reply:
to address the strings in the -exclude part? Just to clarify that this covers the scenario where the feature is excluded, and it's implemented as part of ExcludeStringSet.